src/Sylius/Bundle/ShopBundle/Resources/views/Block/Layout/_afterSecurityWidgetLegacy.html.twig

```diff
@@ -0,0 +1 @@
+{{ sonata_block_render_event('sylius.shop.layout.after_security_widget') }}
```

Comment:
I'm thinking about the way to generalise these calls, creating a template for every legacy function call seems like an overkill.